kombu/transport/redis_cluster.py

```diff
@@ -221,7 +221,8 @@ def _register_BRPOP(self, channel):
                 except Exception as e:
                     logger.error('Error while registering BRPOP', extra={"e": e, "key": conn.key})
 
-        channel._brpop_start()
+        timeout = channel.connection.client.transport_options.get('brpop_timeout', 1)
```

Comment:
what if we do not set the timeout at all? I believe it is the best way to do not use cpu

Reply:
It seems timeout value is mandatory https://redis.io/commands/brpop/

Reply:
We can set timeout to 0 to indefinitely block operation, https://redis.io/commands/blpop/ `A timeout of zero can be used to block indefinitely.`

Reply:
i'll test 0 timeout value after this pr merge

Reply:
Actually what I wonder is that there are any code blocks fall in problem if we set that timeout as zero